### ISSUE TITLE
Sortable tables lack ARIA-sort properties

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#1636]
### Description

When tables allow users to sort the content in a column, the column header should receive an ARIA `columnheader` role and `col` scope property. This will let users of assistive technology know that they are able to sort the information in the table using the table headers. For an example of an accessible sortable  table, see: https://dequeuniversity.com/library/aria/tables/sf-sortable-grid

Two examples of sortable tables that lack these attributes are the table on the '[My Catalog](https://github.com/pressbooks/pressbooks/blob/a93df1e7297b8ad0a4d9339b8d735c2a1ebdc06b/inc/admin/class-catalog-list-table.php)' page and the table on the 'Contributors' page.

**Comment by dannylonglegs** [comment]
https://core.trac.wordpress.org/ticket/46977#ticket

**Comment by mcgratay** [comment]
Sortable tables still need aria-sort property, not just "col" scope and "columnheader" role. 

Distinction needs to be clear between sortable columns and non-sortable columns. See below for example. 

![Screen Shot 2019-04-24 at 2 08 18 PM](https://user-images.githubusercontent.com/39064028/56682853-6f4fac80-669a-11e9-9f1e-372ce0266f95.png)

**Comment by mcgratay** [comment]
Resolved 👍
